pandas/tseries/resample.py

```diff
@@ -652,9 +652,6 @@ def _convert_obj(self, obj):
             # Cannot have multiple of periods, convert to timestamp
             self.kind = 'timestamp'
 
-        if not len(obj):
-            self.kind = 'timestamp'
-
```

Comment:
the ``kind`` argument is quite tricky. We really should remove it, but that's a whole other project. The issue is that some of the routines are expecting ``DatetimeIndex`` so we give it to them even though we are a ``PeriodIndexResampelr``. This all existed before the changed in 0.18.0, but was much trickier to deal with as it was function based. So should be possible to fix now.